src/demo/index.php

```diff
@@ -65,11 +65,17 @@ function gtag() { dataLayer.push(arguments); }
                 <label for="size">Font size</label>
                 <input class="param" type="number" id="size" name="size" placeholder="20" value="20">
 
-                <label for="center">Centered</label>
+                <label for="center">Horizontally Centered</label>
                 <select class="param" id="center" name="center" value="false">
                     <option>false</option>
                     <option>true</option>
                 </select>
+                
+                <label for="center">Vertically Centered</label>
```

Comment:
The long labels cause it to be very difficult to use on mobile since it shrinks the input boxes.

Maybe we could have a field called "alignment" and have options for "top left", "top center", "middle left", "middle center" and have the JS sort it out?

Perhaps using icons to shorten the text could help.

If it's too much to come up with a neat solution, don't worry about it.

![image](https://user-images.githubusercontent.com/20955511/116791825-8b765780-aac5-11eb-8ec1-29a6e716a185.png)

Reply:
Setting a min-width on the parameter input fields might work too since that would force it to wrap the text onto 2 lines.

Reply:
Very good point! I didn't consider mobile.

I really like the min-width approach. I just looked at it locally and it really does the job. Let me know what you think. Great idea 👍🏼